MekHQ/resources/mekhq/resources/Mission.properties

```diff
@@ -42,20 +42,20 @@ AtBLanceRole.UNASSIGNED.text=Unassigned
 AtBLanceRole.UNASSIGNED.toolTipText=The lance is not currently assigned to combat duties.
 
 # AtBMoraleLevel Enum
-AtBMoraleLevel.BROKEN.text=Broken
-AtBMoraleLevel.BROKEN.toolTipText=The unit's morale has broken, and its men are in full retreat.
-AtBMoraleLevel.VERY_LOW.text=Very Low
-AtBMoraleLevel.VERY_LOW.toolTipText=The unit is on precipice of breaking, their leadership barely holding the unit together.
-AtBMoraleLevel.LOW.text=Low
-AtBMoraleLevel.LOW.toolTipText=The unit is demoralized, but is still fighting cohesively.
-AtBMoraleLevel.NORMAL.text=Normal
-AtBMoraleLevel.NORMAL.toolTipText=The unit is ready to fight.
-AtBMoraleLevel.HIGH.text=High
-AtBMoraleLevel.HIGH.toolTipText=The unit is ready and glad to fight.
-AtBMoraleLevel.VERY_HIGH.text=Very High
-AtBMoraleLevel.VERY_HIGH.toolTipText=The unit is dedicated to their cause.
-AtBMoraleLevel.UNBREAKABLE.text=Unbreakable
-AtBMoraleLevel.UNBREAKABLE.toolTipText=The unit's trust in their leadership and dedication to their cause is nigh-on unbreakable, and they look forward to fighting their enemies.
+AtBMoraleLevel.ROUTED.text=Routed
```

Comment:
I opted to rename the levels for three reasons:

- to add distance from official morale rules, so there can be no confusion that this is unofficial
- to better reflect how we're currently using these morale levels, as not just a measurement of enemy mental wellbeing, but also material capacity to fight back.
- with some of the other user-facing terminology changes, this helps give the illusion of fighting a persistent enemy